assets/src/components/form/select/model-selects/editor-select.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Internal imports
+ */
+import * as modelSelect from '../model-select';
+
+/**
+ * External imports
+ */
+import { Component } from '@wordpress/element';
+import { BaseControl, withInstanceId } from '@wordpress/components';
+import { PropTypes } from 'prop-types';
+
+/**
+ * ModelSelect Component wrapped in a BaseControl component.
+ */
+class EditorSelect extends Component {
```

Comment:
> I think doing the conversion will also reveal that we don't really need a lot of what you've done here.

not sure what you are referring to specifically...
I've specified the props because they are different than what `BaseControl` control uses and I also need to apply some modifications, such as setting the `id` as well as the `'aria-describedby'` propso as far as my understanding goes... I can't just pass things straight through.

Reply:
Ya you're right, we'll need some of what you need here.  But there's still some things that wouldn't bee needed here.  Since I'm working on the HOC that will replace this, you'll see the changes in it (but there likely will still be much I'll keep).

Reply:
It looks like you've left this file in when its no longer needed because of the `withBaseControl` HOC I added.